demos/utils.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Forced memory load.
+void ForceRead(const void *p);
+
+// Flush a memory interval from cache.
```

Comment:
```
// Flush a memory interval from cache. This is typically done in order to set up for extracting via Flush+Reload.
```

Something like that? I'm not super familiar with how you use this, so I don't know if there is other info a newbie might want to know about this util function. Feel free to add that too.

Reply:
This has a different purpose - that's for inducing speculative execution. We don't use this in the Flush+Reload scheme. Comment extended.